src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.cs

```diff
@@ -78,8 +93,9 @@ protected override void OnApplyTemplate()
 		_foregroundPaintContext.ShadowHost.Opaque = false;
 #endif
 
-		_canvas.Children.Insert(0, _backgroundPaintContext.ShadowHost);
-		_canvas.Children.Add(_foregroundPaintContext.ShadowHost);
+		_currentContent = (FrameworkElement?)_contentPresenter?.Content;
+		_canvas?.Children.Insert(0, _backgroundPaintContext.ShadowHost);
+		_canvasInner?.Children.Insert(0, _foregroundPaintContext.ShadowHost);
```

Comment:
we throw if the value is `null`, do we need the `?` operator?
```suggestion
		_canvas.Children.Insert(0, _backgroundPaintContext.ShadowHost);
		_canvasInner.Children.Insert(0, _foregroundPaintContext.ShadowHost);
```